packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/MappingTestState.ts

```diff
@@ -695,8 +700,12 @@ export class MappingTestState {
     this.test.setAssert(this.assertionState.assert);
   }
 
-  setExecutionPlan = (val: object | undefined): void => {
+  setExecutionPlan = (
+    val: object | undefined,
+    metaVal: ExecutionPlan | undefined,
+  ): void => {
     this.executionPlan = val;
```

Comment:
Need to put executionPlan, executionPlanMeta inside the executionPlanState (mentioned by @MauricioUyaguari )